be/src/common/daemon.cpp

```diff
@@ -396,6 +397,14 @@ void Daemon::wg_mem_used_refresh_thread() {
     }
 }
 
+void Daemon::be_proc_monitor_thread() {
+    while (!_stop_background_threads_latch.wait_for(
+            std::chrono::milliseconds(config::be_proc_monitor_interval_ms))) {
+        BeProcMonitor bpm;
```

Comment:
should we create a object each while loop，better the `get_be_thread_info()` should be static